### PR TITLE
All HW3 files are ready for review

### PR DESCRIPTION
**Pull request by evgenii-liashenko**
I have completed the third homework and I am looking forward to your feedback.
Just in case, please note that at the time this pull request was opened my second homework had not been reviewed yet.

### CODE REVIEW

File: src/test/java/ru/training/at/hw3/utils/webdriver/DriverConfigurator.java [hunk 1]
```diff
@@ -0,0 +1,25 @@
+package ru.training.at.hw3.utils.webdriver;
+
+import io.github.bonigarcia.wdm.WebDriverManager;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.chrome.ChromeDriver;
+import java.util.concurrent.TimeUnit;
+
+public class DriverConfigurator {
```
Comment (DmitryKhodakovsky) [inline src/test/java/ru/training/at/hw3/utils/webdriver/DriverConfigurator.java:8]: Do you try implement **Singleton** design pattern here?

**Review by DmitryKhodakovsky — APPROVED** [review]
HW accepted

**Comment by DmitryKhodakovsky** [comment]
Please resolve comments and merge to the **master** branch

File: src/test/java/ru/training/at/hw3/utils/webdriver/DriverConfigurator.java (hunk 1 above)
Comment (evgenii-liashenko) [reply src/test/java/ru/training/at/hw3/utils/webdriver/DriverConfigurator.java:8]: No, I had no intentions to implement this pattern. The task requires to "correctly manage the place where you will create a driver instance", so I just studied the code from the lecture and decided to do something similar in my homework.

**Pull request merged**